src/style.css

```diff
@@ -0,0 +1,40 @@
+* {
+  margin: auto;
+}
+
+.important,
+.interesting,
+.surprising,
+#header,
+#footer {
+  width: 300px;
+  height: 100px;
+}
+
+.important:hover,
+.interesting:hover,
+.surprising:hover,
+#header:hover,
+#footer:hover {
```

Comment:
but you should delete this declaration

Reply:
This declaration makes height change when hovered over. If I delete this declaration the solution will not be in compliance with the requirements. Or I misunderstood smth?

Reply:
Oh, sorry i missed hover